### DISCUSSION TITLE
plugin_lines_sections is an invalid input?

### DISCUSSION THREAD

**Discussion by revilowaldow** [lowlighter/metrics#1166] [Q&A] [answered]
Got the below warning on my metrics:
![image](https://user-images.githubusercontent.com/52298102/182001988-8bf4bb18-3216-4f1e-9f3e-6f95c9e79410.png)
However the documentation suggests that this input option exists:
https://github.com/lowlighter/metrics/blob/39955fa957ddc3cb13fcc4b6431d751615e9b6a6/source/plugins/lines/README.md?plain=1#L59

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

It may not be very noticeable, but there's a "✨ On master/main" label.

I agree it isn't very clear, but basically it means that these options are currently only on the ˋmaster/main` branches

These options will officially be released on ˋlatest` branch with next version (#1163)

You can chose to switch branches to test new features when they're merged, though sometimes the master branch may break occasionally 

Sorry for the confusion 😅
Hope it helps

**Reply by revilowaldow** [reply]
Ahhh understood, thank you. That makes a lot of sense.
